src/main/java/seedu/address/ui/ThemeManager.java

```diff
@@ -12,6 +12,8 @@
  */
 public class ThemeManager {
 
+    private static ThemeManager instance;
```

Comment:
Singleton pattern? Better write in this class that `ThemeManager` is a singeleton.